week-5/InClass/A-dom-manipulation/exercise.js

```diff
@@ -39,12 +80,39 @@ Task 4
 
 When a user clicks the 'Add some text' button, a new paragraph should be added below the buttons that says "Read more below."
 */
+let button = document.querySelector("#addTextBtn")
+let jumbotron = document.querySelector(".jumbotron")  
 
+button.addEventListener("click", function() {
+    let paragraph = document.createElement("p");
+    paragraph.innerText = "Read more below";
+    jumbotron.appendChild(paragraph);
+});
 
 
 /*
 Task 5
 ======
 
 When the 'Larger links!' button is clicked, the text of all links on the page should increase.
-*/
\ No newline at end of file
+*/
+
+let paragraph = document.querySelector("#largerLinksBtn");
+paragraph.addEventListener("click", largeLink);
+function largeLink() {
+    let links = document.querySelectorAll("a");
+    // for (i=0; i<links.length; i++) {
+    //     links[i].style.fontSize = "200%";
+    // }
+
+    links.forEach(element => element.style.fontSize = "200%");
+    
+    console.log(links);
```

Comment:
You can normally remove these `console.log()` calls when you submit your homework